app/models/extractors/question_extractor.rb

```diff
@@ -1,10 +1,13 @@
 module Extractors
   class QuestionExtractor < Extractor
+    class MissingAnnotation < StandardError; end
+
     validates :task_key, presence: true
+    validates :if_missing, presence: true, inclusion: {in: ["error", "ignore"]}
```

Comment:
does this mean you have to backfill all the old extractors to have this value in the config?

Reply:
No, the accessor below has a default.

Reply:
won't this mean that older extractors without the config will be invalid from now on? and won't update / save?

Reply:
The validation uses the accessor method to get the value, so they're still valid.

Reply:
my mistake - got it now, nice.